ice_in

```diff
@@ -1,450 +1,156 @@
-  nblyr = 3
+  nblyr = 1
   ncat = 5
   nfsd = 1
   nilyr = 4
   nslyr = 1
-  orca_halogrid = .false.
-  use_bathymetry = .false.
 /
 &tracer_nml
-  n_aero = 3
-  n_algae = 0
-  n_dic = 0
-  n_doc = 0
-  n_don = 0
-  n_fed = 0
-  n_fep = 0
-  n_iso = 3
-  n_zaero = 0
-  restart_aero = .false.
-  restart_age = .false.
-  restart_fsd = .false.
-  restart_fy = .false.
-  restart_iso = .false.
-  restart_lvl = .false.
-  restart_pond_lvl = .false.
-  restart_pond_topo = .false.
-  restart_snow = .false.
-  tr_aero = .false.
-  tr_fsd = .false.
-  tr_fy = .true.
-  tr_iage = .true.
-  tr_iso = .false.
-  tr_lvl = .true.
-  tr_pond_lvl = .true.
-  tr_pond_topo = .false.
-  tr_snow = .false.
 /
 &thermo_nml
-  a_rapid_mode = 0.5e-03
-  aspect_rapid_mode = 1.0
-  conduct = "MU71"
-  dsdt_slow_mode = -1.5e-07
-  kitd = 1
-  ktherm = 2
-  phi_c_slow_mode = 0.05
-  phi_i_mushy = 0.85
-  rac_rapid_mode = 10
+  dsdt_slow_mode = -5e-08
 /
 &dynamics_nml
-  advection = "remap"
-  alphab = 20.0d0
-  arlx = 300.
-  brlx = 300.
-  cf = 17.0
-  coriolis = "latitude"
-  cstar = 20.0d0
-  e_plasticpot = 2.0
-  e_yieldcurve = 2.0
   k1 = 8.0d0
-  k2 = 15.0d0
-  kdyn = 1
-  krdg_partic = 1
-  krdg_redist = 1
-  kridge = 1
-  kstrength = 1
-  ktens = 0.0
-  ktransport = 1
-  mu_rdg = 4.0
-  ndte = 120
-  pstar = 2.75d4
-  revised_evp = .false.
-  seabed_stress = .false.
-  seabed_stress_method = "LKD"
-  ssh_stress = "coupled"
-  threshold_hw = 30.0d0
 /
 &shortwave_nml
-  ahmax = 0.3
-  albedo_type = "default"
-  albicei = 0.45
-  albicev = 0.75
-  albsnowi = 0.73
+  ahmax = 0.1
+  albicei = 0.44
+  albicev = 0.86
+  albsnowi = 0.7
   albsnowv = 0.98
-  dt_mlt = 1.50
+  dt_mlt = 1.0
   kalg = 0.0
-  r_ice = 0.0
-  r_pnd = 0.0
-  r_snw = 1.25
-  rsnw_mlt = 1500.
-  shortwave = "dEdd"
+  r_snw = 0.0
   sw_dtemp = 0.01d0
   sw_frac = 1.0d0
   sw_redist = .true.
 /
 &ponds_nml
   dpscale = 1.0e-3
-  frzpnd = "cesm"
-  hp1 = 0.01
-  hs0 = 0.0
-  hs1 = 0.03
-  pndaspect = 0.8
-  rfracmax = 0.85
-  rfracmin = 0.15
+  frzpnd = "hlid"
+  rfracmax = 1.0
 /
 &snow_nml
-  drhosdwind = 27.3
-  rhosmax = 450.0
-  rhosmin = 100.0
-  rhosnew = 100.0
-  rsnw_fall = 100.0
-  rsnw_tmax = 1500.0
-  snw_T_fname = "unknown"
-  snw_Tgrd_fname = "unknown"
-  snw_aging_table = "test"
-  snw_drdt0_fname = "unknown"
-  snw_filename = "unknown"
-  snw_kappa_fname = "unknown"
-  snw_rhos_fname = "unknown"
-  snw_tau_fname = "unknown"
-  snwgrain = .false.
-  snwlvlfac = 0.3
-  snwredist = "none"
-  use_smliq_pnd = .false.
-  windmin = 10.0
 /
 &forcing_nml
-  atmbndy = "default"
-  calc_strair = .true.
-  calc_tsfc = .true.
   emissivity = 0.95d0
-  fbot_xfer_type = "constant"
-  formdrag = .false.
   highfreq = .true.
-  ice_ref_salinity = 4.0d0
-  l_mpond_fresh = .false.
-  natmiter = 5
-  nfreq = 25
-  restart_coszen = .false.
-  rotate_wind = .true.
-  saltflux_option = "prognostic"
-  tfrz_option = "mushy"
+  tfrz_option = "linear_salt"
   update_ocn_f = .true.
   ustar_min = 0.005
```

Comment:
Let's use
```suggestion
  ustar_min = 0.0005
```
for now to match access-om2. This is also used in https://github.com/CICE-Consortium/CICE/blob/b14cedfaed8b81500fc5422cfc44b6d80e5893ef/configuration/scripts/ice_in#L253. Also see https://bb.cgd.ucar.edu/cesm/threads/ustar_min-default-documentation.9059/